doc/htmldoc/models/index.rst

```diff
@@ -5,8 +5,32 @@ Model directory
 ===============
 
 
+What type of model?
+-------------------
+
 NEST has over 100 models, choose an option for finding the one you need!
 
+.. grid::
+
+   .. grid-item-card::
+      :columns: 4
+      :link: network_models
+      :link-type: ref
+
+      **Network Models**
+
+
+
+   .. grid-item-card::
+      :columns: 4
+      :link: model_selector
+      :link-type: ref
+
+      **Neurons, synapses, and devices**
```

Comment:
Instead of these cards, I suggest to start the page with the section "What we mean by models" that is currently further down (https://nest-simulator--3428.org.readthedocs.build/en/3428/models/index.html#what-we-mean-by-models). This section should also mention devices.

Reply:
@jessica-mitchell I think for a new user it might be unclear that the model selector does not refer to network models after showing the cards of both. Yes, the "What's the difference?" already helps. In addition, I suggest to extend "Select a tag to display corresponding models." to "Select a tag to display corresponding neuron, synapse or device models."